### PR TITLE
[9.0] extract analytic_department

### PR DESCRIPTION
**Pull request by yvaucher**
Move https://github.com/OCA/department/pull/54 here

Depends on https://github.com/OCA/project/pull/170

### CODE REVIEW

File: analytic_department/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# © 2011 Joël Grand-Guillaume (Camptocamp)
+# © 2013 Daniel Reis (Camptocamp)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Analytic Department Categorization",
+    "version": "8.0.1.0.0",
+    "author": "Camptocamp, Daniel Reis,Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "category": "Generic Modules/Projects & Services",
+    "description": """\
```
Comment (yvaucher) [inline analytic_department/__openerp__.py:11]: to move in README.rst

File: analytic_department/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# © 2011 Joël Grand-Guillaume (Camptocamp)
+# © 2013 Daniel Reis (Camptocamp)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Analytic Department Categorization",
+    "version": "8.0.1.0.0",
```
Comment (dreispt) [inline analytic_department/__openerp__.py:7]: version 9

**Comment by dreispt** [comment]
IMO this is better proposed to OCA/contract

**Comment by yvaucher** [comment]
Same comment as in #170, what about https://github.com/oca/account-analytic ?

**Comment by yvaucher** [comment]
Moved in https://github.com/OCA/account-analytic/pull/56

**Pull request closed without merging**